specification/elasticsan/resource-manager/Microsoft.ElasticSan/preview/2022-12-01-preview/examples/PrivateEndpointConnections_Create_MaximumSet_Gen.json

```diff
@@ -25,22 +25,22 @@
     "200": {
       "body": {
         "properties": {
-          "provisioningState": "Invalid",
+          "provisioningState": "Succeeded",
           "privateEndpoint": {
-            "id": "kywwthb"
+            "id": "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/privateEndpoints/{privateEndpointName}"
```

Comment:
The input and output should be aligned, please also use `{subscriptionId}`, `{resourceGroupName}`, `{privateEndpointName}` in the input parameters.